Jenkinsfile

```diff
@@ -46,6 +46,11 @@ git merge ${merge_opts} '${commit}' -m '${commit_msg}'
 """)
 }
 
+// https://issues.jenkins-ci.org/browse/JENKINS-37069
+def fix_workspace_label() {
+    sh "chcon -Rt svirt_sandbox_file_t '${env.WORKSPACE}'*"
```

Comment:
We really have our hands tied by the way docker plugin executes the `docker run` command:

https://github.com/jenkinsci/docker-workflow-plugin/blob/master/src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java#L110

The problem is not so much about the workspace, we could copy it to another directory and mount it with the `:Z` suffix. Jenkins also mounts another directory (that's why we have the `*` at the end) to keep some control files (logs, pid, etc.) and the mount flags for that are also hard-coded.